reporters/dogstatsd/handlers.go

```diff
@@ -135,6 +134,5 @@ func GaugeHandler(
 	if err != nil {
 		return err
 	}
-	c.Gauge(event, gauge, tags, 1)
-	return nil
+	return c.Gauge(event, gauge, tags, 1)
```

Comment:
Can you wrap this and other errors with `fmt.Errorf`?

Reply:
Wrapping gauge errors with fmt.Errorf